config/routes.rb

```diff
@@ -95,9 +95,9 @@
 
   resources :invitations, only: [:show, :create, :destroy], param: :invitation_slug do
     member do
-      post :accept, action: :update
-      post :decline, action: :update, decline: true
-      post :resend, action: :resend
+      get :accept
```

Comment:
Yeah I puzzled over that a bit. The way teammate invitations work, the accept & decline links are right there in the email, so I assume whoever did it originally used GETs for that reason. The speaker invitation email only contains a link to the invitation show page, and so the accept & decline links can be whatever, and so were done with POSTs, which isn't too bad. 

I made them GETs here for consistency with the teammate invitation routes, and also for the convenience of simply redirecting back to the accept url after sign/signup. I didn't look into whether the redirect could be done with a different verb.